tests/perf/run_perf_tests.py

```diff
@@ -230,14 +281,19 @@ def get_result(self, test_number, daemon_name, qps_value, time_value):
             max_latency = latency_values[12]
             newline = file_raws[5].replace('\n', ' ')
             ratio = newline.split(' ')[34]
-            print(" [ Ratio="+ratio+", MaxLatency="+max_latency+"]")
+            if len(file_raws) > 8:
+                error = file_raws[8]
+                print(" [ Ratio="+ratio+", MaxLatency="+max_latency+ " Error: " + error +"]")
+            else:
+                error = ""
+                print(" [ Ratio="+ratio+", MaxLatency="+max_latency+"]")
             threads = os.popen("ps -efL | grep tg | grep bin | wc -l").read().replace('\n', ' ')
         finally:
             file.close()
 
         self.test_report.write_test_report(daemon_name, test_number, threads, qps_value, time_value, min_latency, latency_values[7], latency_values[8], \
-                                           latency_values[9], latency_values[10], latency_values[11], max_latency, ratio)
-        os.system("/bin/rm " + test_report_filename)
+                                           latency_values[9], latency_values[10], latency_values[11], max_latency, ratio, error)
+        #os.system("/bin/rm " + test_report_filename)
```

Comment:
Shall we remove this line or still need it someway?

Reply:
oops